docs/release-notes/2024-12-20-new-sponsored-count-parameter.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "FastStore: Improved search query with the `sponsoredCount` parameter"
+slug: "2024-12-20-new-sponsored-count-parameter"
+hidden: false
+type: "added"
+createdAt: "2024-12-20T00:00:00.219Z"
+excerpt: "FastStore projects now include a new parameter to improve the management of sponsored product search results."
+---
+
+We've added a new parameter, `sponsoredCount`, to the search query. This parameter gives more control over the number of sponsored products returned in your store’s search results.
+
+> ⚠ Extension API users: If you are using the [`ClientProductGallery`](https://developers.vtex.com/docs/guides/faststore/api-extensions-extending-queries-using-fragments#clientproductgalleryquery) fragment, this change may cause errors. Make sure your implementation is updated to handle the `sponsoredCount` parameter accordingly by following the instructions in the [next section](#api-extension-users).
```

Comment:
```suggestion
> ❗️ Extension API users: If you are using the [`ClientProductGallery`](https://developers.vtex.com/docs/guides/faststore/api-extensions-extending-queries-using-fragments#clientproductgalleryquery) fragment, this change may cause errors. To prevent issues, ensure your implementation is updated to handle the `sponsoredCount` parameter by following the instructions in the [API Extension Users](#api-extension-users) section below.
```